scapy/contrib/automotive/daimler/definitions.py

```diff
@@ -0,0 +1,3400 @@
+# flake8: noqa: E501
+
+# This file is part of Scapy
+# See http://www.secdev.org/projects/scapy for more information
+# Copyright (C) Nils Weiss <nils@we155.de>
+# This program is published under a GPLv2 license
+
+# scapy.contrib.description = Daimler specific definitions for UDS
+# scapy.contrib.status = loads
+
+
+from scapy.contrib.automotive.uds import UDS_RDBI
+
+UDS_RDBI.dataIdentifiers[0x0000] = "Read all stored UID from Kleer-Module"
+UDS_RDBI.dataIdentifiers[0x0002] = "Freeze Frame Diagnostic Trouble Code"
+UDS_RDBI.dataIdentifiers[0x0003] = "Kalibrierung"
+UDS_RDBI.dataIdentifiers[0x0004] = "Calculated Load Value"
+UDS_RDBI.dataIdentifiers[0x0005] = "Engine Coolant Temperature"
+UDS_RDBI.dataIdentifiers[0x0006] = "Status FIN-Verriegelung"
+UDS_RDBI.dataIdentifiers[0x0007] = "Service Sensorjustage Fortschrittsanzeige"  	 # or Zustand ELV Fahrt Verriegelung 1 ELV Verrieglung
+UDS_RDBI.dataIdentifiers[0x0008] = "Status Sensor Justage "
+UDS_RDBI.dataIdentifiers[0x0009] = "EOL Sensorjustage Fortschrittsanzeige"  	 # or Kraftstoffdruck Sollwert Sollwert Aktuell, Kraftstoffdruck Sollwert
+UDS_RDBI.dataIdentifiers[0x000a] = "AD Werte Batteriespannung"
+UDS_RDBI.dataIdentifiers[0x000b] = "EKP Motorspannung PRES Spannung"  	 # or Batteriespannung PRES Spannung, Temperatursensor PRES Temperatur, Batteriespannung, KSD Sensor Signal PRES Kraftstoffdruck
```

Comment:
codespell returns a lot of errors for this file. See for example https://travis-ci.com/secdev/scapy/jobs/269510025#L662

You might want to disable spellcheck on this file at https://github.com/secdev/scapy/blob/master/tox.ini#L160

Reply:
A lot of this definitions are parsed from random files. Unfortunately its a mix of German and Englisch, but there are no better definitions available.

Reply:
Done. Thanks for the hint.